paper/src/main/java/rocks/gravili/notquests/paper/conversation/ConversationManager.java

```diff
@@ -20,12 +20,8 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.UUID;
+import java.util.*;
```

Comment:
not sure a wildcard import for the entire java.util class is a good idea here

Reply:
IntelliJ reduces automatically 6 imports into only one with a wildcard. Same as in the QuestPlayerManager class.